### PR TITLE
Port basic text dataset

### PR DESCRIPTION
**Pull request by gmrukwa**
Minimalistic dataset for our current needs.

Additionally, I've placed some badges in the readme.

### CODE REVIEW

File: spdata/reader.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""Various methods for reading data"""
+
+import numpy as np
+
+import spdata.types as ty
+
+
+def _parse_metadata(line: str) -> (int, int):
```
Comment (Daroslaw) [inline spdata/reader.py]: We don't need label parsing for now (and preemptively Z coordinate)?

File: spdata/types.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+"""Reusable types across package"""
+
+
+import numpy as np
+
+
+class Dataset:
```
Comment (Daroslaw) [inline spdata/types.py]: Same here - why are we omitting Z and labels

**Review by Daroslaw — CHANGES_REQUESTED** [review]
If you convince me that adding additional parsing for Z coordinate and label is redundant enough to not implement it now, or if you actually implement it, I will approve this PR.

File: spdata/reader.py (hunk 1 above)
Comment (gmrukwa) [reply spdata/reader.py]: That's why I asked you for review - I was not fully sure with this.

Comment (Daroslaw) [reply spdata/reader.py]: IMO if this we have a port of some other code in C#, then 100% of its functionalities should be provided in final code.

Comment (gmrukwa) [reply spdata/reader.py]: Update incoming...

File: spdata/types.py (hunk 2 above)
Comment (gmrukwa) [reply spdata/types.py]: Not omitting anymore.

**Review by Daroslaw — APPROVED** [review]

**Comment by Daroslaw** [comment]
Now it just needs to build 😛

**Comment by gmrukwa** [comment]
Yeah, there's something strange there for *conda*: https://ci.appveyor.com/project/gmrukwa/spectre-data/build/job/379ditlugpf316ao
Also, I am splitting `Dataset` to make it go through *BetterCodeHub*. This may get interesting.

**Comment by gmrukwa** [comment]
Mystery partially solved: [build 0.0.0.11](https://ci.appveyor.com/project/gmrukwa/spectre-data/build/0.0.0.11/job/alhujadryma432dk).

**Comment by gmrukwa** [comment]
[Data access layer for Python](https://trello.com/c/Ie7Lc0zL/350-data-access-layer-for-python)

**Review by mg6 — APPROVED** [review]
After some refactoring, seems fine now. We need to investigate further optimization possibilities in the future, especially for data loading method.

Nice work!

**Pull request merged**